src/components/CheckboxInput/CheckboxInput.js

```diff
@@ -114,17 +121,35 @@ export const CheckboxInput = ({
 }
 
 CheckboxInput.propTypes = {
+  /** low level prop used only by the form engine */
   formTouched: PropTypes.bool,
-  name: PropTypes.string.isRequired, // must be unique
+  /** Name of the field */
```

Comment:
We've learned that this sort of commenting helps with the output react-styleguidist generates in the EDS for props. We didn't do this in a lot of places, so I'm trying to just add them in as I encounter files organically.